### PR TITLE
Added C++ implementation

### PR DESCRIPTION
**Pull request by NazarPolonsky**
Nazar Holickov IT-42

### CODE REVIEW

File: C++/1-for.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+using namespace std;
+
+void For()
+{
+	for (int i = 0; i < 10; i++)
+	{
+		cout << i << endl;
+	}
+	_getch();
```
Comment (nechaido) [inline C++/1-for.cpp]: I do not like the idea to use `_geth()` in here, if you think it is vital to get user input before ending the program - you could write an example runner.

File: C++/1-for.cpp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#include "stdafx.h"
+#include <iostream>
+#include "conio.h"
+
+using namespace std;
+
+void For()
+{
```
Comment (nechaido) [inline C++/1-for.cpp]: We prefer to use [google c++ style guide](https://google.github.io/styleguide/cppguide.html), so the brace should stay on the same line.

File: C++/1-for.cpp [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	for (int i = 0; i < 10; i++)
+	{
```
Comment (nechaido) [inline C++/1-for.cpp]: Ditto.

File: C++/1-for.cpp [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#include "stdafx.h"
```
Comment (nechaido) [inline C++/1-for.cpp]: Is is really needed? I see no reason for this file to include `"stdafx.h"`. Moreover it isn't event committed.

File: C++/1-for.cpp [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+#include <iostream>
+#include "conio.h"
+
+using namespace std;
+
+void For()
+{
+	for (int i = 0; i < 10; i++)
```
Comment (nechaido) [inline C++/1-for.cpp]: Indentation must be made with 2 spaces not tabs.

File: C++/8-forEach.cpp [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+#include <iostream>
+#include "conio.h"
+
+using namespace std;
+
+void ForEach()
+{
+	int arr[] = { 4, 2, 7, 8, 1, 6, 9, 5 };
+	int index = 0;
+	for each (int item in arr)
```
Comment (nechaido) [inline C++/8-forEach.cpp]: It is Visual C++ only. Moreover it is [deprecated](https://msdn.microsoft.com/en-US/library/ms177202.aspx).
> Iterates through an array or collection. This non-standard keyword is available in both C++/CLI and native C++ projects. However, its use is not recommended. Consider using a standard Range-based for Statement (C++) instead.

You should have used `for (auto item : arr)`

File: C++/1-for.cpp [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+	{
+		cout << i << endl;
+	}
+	_getch();
+}
```
Comment (aqrln) [inline C++/1-for.cpp]: Missing newline at the end of file.

File: C++/2-while.cpp [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+void While()
+{
+	int index = 0;
+	while (index < 10)
+	{
+		cout << index++ << endl;
+	}
+	_getch();
+}
```
Comment (aqrln) [inline C++/2-while.cpp]: Missing newline at the end of file.

File: C++/3-doWhile.cpp [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+{
+	int index = 0;
+	do
+	{
+		cout << index++ << endl;
+
+	} while (index < 10);
+
+	_getch();
+}
```
Comment (aqrln) [inline C++/3-doWhile.cpp]: Missing newline at the end of file.

File: C++/4-forIn.cpp [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+	int arr[] = { 4, 7, 21, 4, 3, 90, 5 };
+
+	for (int i = 0; i < _countof(arr); i++)
+	{
+		int value = arr[i];
+		cout << i << ' ' << value << endl;
+	}
+
+	_getch();
+}
```
Comment (aqrln) [inline C++/4-forIn.cpp]: Ditto

File: C++/6-break.cpp [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+	{
+		cout << "Hello ";
+		if (flag) break;
+		cout << "World!";
+	}
+
+	cout << "There!";
+
+	_getch();
+}
```
Comment (aqrln) [inline C++/6-break.cpp]: Ditto

File: C++/7-continue.cpp [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+{
+	for (int i = 0; i < 5; i++)
+	{
+		cout << "Hello ";
+		if (i != 4) continue;
+		cout << "World!" << endl;
+	}
+
+	_getch();
+}
```
Comment (aqrln) [inline C++/7-continue.cpp]: Ditto

File: C++/8-forEach.cpp [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+	{
+		cout << index << ' ' << item << endl;
+		index++;
+	}
+
+	_getch();
+}
```
Comment (aqrln) [inline C++/8-forEach.cpp]: Ditto

**Comment by nechaido** [comment]
Closed due to inactivity.

**Pull request closed without merging**